Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -25,14 +26,26 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _resizeDirection = GetResizeDirection();
             _resizeBehavior = GetResizeBehavior();
 
+            GridSplitterGripper gripper;
+
             // Adding Grip to Grid Splitter
             if (Element == default(UIElement))
             {
-                var element = new GridSplitterGripper(
+                gripper = new GridSplitterGripper(
                     _resizeDirection,
                     GripperForeground);
-                Element = element;
             }
+            else
+            {
+                var content = Element;
+                Element = null;
+                gripper = new GridSplitterGripper(content, _resizeDirection);
+            }
+
+            Element = gripper;
+
+            gripper.OnKeyboardHorizontalMoveAction = (movement) => HorizontalMove(movement);
```

Comment:
The gripper is tied directly to the splitter. It says so in the name of the control. An Action allows it to be changed therefore breaking the functionality. 
I would prefer either

1. Require Gridsplitter in the ctor
2. Fire an event from the gripper